frontend/public/index.html

```diff
@@ -1,5 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```

Comment:
looks like you've commited a file with a conflict. see if you can resolve this?

Reply:
@bheadwhite Issue resolved, we had another team member make some changes that were merged in before I realized